integration_test/third_party_apps_data/applications/jvm/sles/install

```diff
@@ -12,7 +12,11 @@ if [[ "${ID}" == opensuse-leap && "${VERSION_ID}" == 15.[01] ]]; then
 fi
 sudo zypper -n refresh
 
-sudo zypper -n install java-1_8_0-openjdk java-1_8_0-openjdk-devel
+if [[ "$(uname -m)" == aarch64 ]]; then
+  sudo zypper -n install java-11-openjdk java-11-openjdk-devel
```

Comment:
is there anything useful we could leave as a comment here?

Reply:
@martijnvans Sure, I can add something here (the arm64 machines come with java 11 vs java 8)